README.md

```diff
@@ -33,6 +33,14 @@ After using [CyberChef](https://gchq.github.io/CyberChef/) to experiment and fin
 - In a browser, navigate to `localhost:3000` to see usage documentation.
 
 
+### Docker
+A Docker image can be built, then run by doing the following:
+
+- `git clone https://github.com/gchq/CyberChef-server`
+- `cd CyberChef-server`
+- `sudo docker build -t mylocalrepo/cyberchef-server .`
+- `sudo docker run -dit --name=cyberchef-server -p 0.0.0.0:3000:3000 mylocalrepo/cyberchef-server`
```

Comment:
Please confirm, but I think `-p 3000:3000` would suffice here.

Reply:
Yes, I've adjusted it.  It would work for local access, but if you wanted to expose it to another service you may need to open it up with 0.0.0.0, adjust local FW rules, etc.